google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/ConnectionWorkerTest.java

```diff
@@ -97,11 +97,13 @@ public void testMultiplexedAppendSuccess() throws Exception {
           StreamWriter.newBuilder(TEST_STREAM_1, client)
               .setWriterSchema(createProtoSchema("foo"))
               .setLocation("us")
+              .setTraceId("some_head:trace_id_1")
               .build();
       StreamWriter sw2 =
           StreamWriter.newBuilder(TEST_STREAM_2, client)
               .setWriterSchema(createProtoSchema("complicate"))
               .setLocation("us")
+              .setTraceId("some_head:trace_id_2")
```

Comment:
Do you also have a test where the StreamWriter traceId is left as NULL?

Reply:
Done